aldryn_config.py

```diff
@@ -4,8 +4,8 @@
 
 class Form(forms.BaseForm):
 
-    show_all_recipients = forms.CheckboxField('Show all users in form recipients select', required=False)
+    show_all_recipients = forms.CheckboxField('Include only admin users as selectable e-mail recipients instead of everyone.', required=False)
 
     def to_settings(self, data, settings):
-        settings['ALDRYN_FORMS_SHOW_ALL_RECIPIENTS'] = data['show_all_recipients']
+        settings['ALDRYN_FORMS_SHOW_ALL_RECIPIENTS'] = not data['show_all_recipients']
```

Comment:
switching it like this would mean that everyone that has checked the box will now get the reverse behavior and everyone that had left it blank (default) would also get reverse behavior.
This would be quite confusing without some sort of migration.